cmd/traffic/cmd/manager/service.go

```diff
@@ -1028,7 +1147,7 @@ func (s *service) WatchWorkloads(request *rpc.WorkloadEventsRequest, stream rpc.
 	return ww.Watch(ctx, stream)
 }
 
-const agentSessionTTL = 15 * time.Second
+const agentSessionTTL = 1 * time.Minute
```

Comment:
This duration must be slightly longer than the poll time used by the traffic-agent so that we don't risk that it's being garbage collected due to a race.
```suggestion
const agentSessionTTL = 70 * time.Second
 ```